packages/services/.env.example

```diff
@@ -15,3 +15,7 @@ CORS_PROXY=<CORS proxy server for fetching from www.ibm.com, e.g. https://yourpr
 
 # Analytics - scroll
 SCROLL_TRACKING=<boolean to enable scroll tracking, e.g. false>
+
+# Kaltura Video Player
+KALTURA_PARTNER_ID=243342
+KALTURA_UICONF_ID=12905712
```

Comment:
@mkothur shouldn't include the actual numbers in the `.env.example`, should be more of a description like above. We probably should only share these numbers internally and not in the source.

Reply:
Okay @jeffchew